src/components/LastBuildCard.tsx

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+import {
+  Link,
+  Theme,
+  makeStyles,
+  LinearProgress,
+  Box,
+  Typography,
+  CircularProgress,
+} from '@material-ui/core';
+import { InfoCard, StructuredMetadataTable } from '@backstage/core';
+import ExternalLinkIcon from '@material-ui/icons/Launch';
+import { useBuilds } from '../hooks/useBuilds';
+import { getStatusComponent } from './BuildsPage/lib/CITable/CITable';
+import moment from 'moment';
+
+const useStyles = makeStyles<Theme>({
+  externalLinkIcon: {
+    fontSize: 'inherit',
+    verticalAlign: 'bottom',
+  },
+});
+
+const WidgetContent = ({
+  loading,
+  lastRun,
+}: {
+  loading?: boolean;
+  lastRun: any;
+}) => {
+  const classes = useStyles();
+  if (loading || !lastRun) return <LinearProgress />;
+
+  return (
```

Comment:
What do you think about using a StructuredMetadataTable component to display this information?
https://backstage.io/storybook/?path=/story/structured-metadata-table--default
Similar usages:
plugins/jenkins/src/components/Cards/Cards.tsx:43
plugins/github-actions/src/components/Cards/Cards.tsx:59